cli_tools_tests/e2e/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -208,6 +211,38 @@ func runImageImportOSFromImageTest(ctx context.Context, testCase *junitxml.TestC
 	runImportTest(ctx, argsMap[testType], testType, testProjectConfig.TestProjectID, imageName, logger, testCase)
 }
 
+func runImageImportOSFromVHDImageTest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
+
+	suffix := path.RandString(5)
+	imageName := "e2e-test-image-import-os-from-vhd-image-" + suffix
+
+	argsMap := map[e2e.CLITestType][]string{
+		e2e.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-image_name=%v", imageName), "-os=ubuntu-1804",
+			fmt.Sprintf("-source_file=gs://%v-test-image/ubuntu-1804-azure.vhd", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-zone=%v", testProjectConfig.TestZone),
+		},
+		e2e.GcloudBetaProdWrapperLatest: {"beta", "compute", "images", "import", imageName, "--quiet",
+			"--docker-image-tag=latest", "--os=ubuntu-1804", fmt.Sprintf("--project=%v", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-source_file=gs://%v-test-image/ubuntu-1804-azure.vhd", testProjectConfig.TestProjectID),
```

Comment:
should be --source_file other than -source_file

Reply:
It's actually --source-file